### DISCUSSION TITLE
Display larger text and/or numbers

### DISCUSSION THREAD

**Discussion by Doctor-love** [Textualize/textual#2874] [Q&A] [answered]
Greetings,

I would like to display some characters (more specifically, numbers) that are bigger than the "regular" terminal text.
I understand why this would be a challenge (font-size is set by the terminal), but perhaps ASCII-/ANSI-art could be used to represent something like a ["seven-segment display"](https://en.wikipedia.org/wiki/Seven-segment_display). See mock-up below.

Do y'all have any other suggestions for how I may accomplish this with Textual? Keep up the good work! 

![large_num_mock](https://github.com/Textualize/textual/assets/2008673/56d610aa-0f10-41a1-a40c-308d12a77e3f)

**Comment by willmcgugan** [comment, marked as answer]
ascii / unicode art should work quite well. You might want to use [this library](https://pypi.org/project/art/) which is known to work with Textual.

**Reply by TomJGooding** [reply]
Thanks Will, I've also bookmarked that library for future reference!

Before you posted, I had some fun quickly knocking together this Textual app for generating text banners (using FIGlet which looks like a subset of available fonts from the library you've linked):

- https://github.com/TomJGooding/figtext

**Reply by willmcgugan** [reply]
Awesome!

@darrenburns @davep

**Reply by davep** [reply]
Nice!
